docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -47,6 +47,8 @@ apply during code compilation.
  ====================================== ============= ==========================
  Sequential/SIMD Execution Policies     Works with    Brief description
  ====================================== ============= ==========================
+ seq_launch_t                           launch        Creates a sequential
```

Comment:
Is there a reason why the '_t' suffix is used here? It appears unique to that case.

Reply:
All the launch types have them, perhaps we can look into removing them for the following release? It would be a small breaking change.